packages/strapi-plugin-content-manager/test/index.test.e2e.js

```diff
@@ -351,6 +351,29 @@ describe('Content Manager End to End', () => {
       expect(body.message).toBe('Already published');
     });
 
+    test('Unpublish article1, expect article1 to be set to null', async () => {
+      const entry = _.clone(data.articles[0]);
+
+      let { body } = await rq({
+        url: `/content-manager/explorer/application::article.article/unpublish/${entry.id}`,
+        method: 'POST',
+      });
+
```

Comment:
Can you add `data.articles[0] = body;` here too please?

Reply:
Nice catch